resources/schemas.json

```diff
@@ -864,10 +864,6 @@
                     "nature": {
                         "type": "string",
                         "description": "The file's nature"
-                    },
-                    "submitter": {
-                        "type": "string",
-                        "description": "The SS58 address of the file submitter"
```

Comment:
Will you remove this from frontend or do I do it?

Reply:
It has never been added, from what I can see.